### ISSUE TITLE
Prop "story" marked as required - warning

### ISSUE THREAD

**Issue by evangeloszotos** [gvaldambrini/storybook-router#43]
Issue:

### Which version are you using?
1.0.7
### Are you using storybook-router with a react based project or a vue one?
react
### Please describe the problem:
A warning is displayed that the component StoryRouter needs the story prop. Although the example seem to work fine. I had look inside the code and indeed, the story property is marked as required, but it is not used in the StoryRouter component. The StoryRouterDecorator is not providing a story prop to StoryRouter either.
Maybe just to delete the declaration in StoryRouter.propTypes may fix the issue.

Warning:
![image](https://user-images.githubusercontent.com/33234890/64604161-f12e4100-d3c1-11e9-8d26-5322297d5461.png)

In Code:
![image](https://user-images.githubusercontent.com/33234890/64604225-128f2d00-d3c2-11e9-9b38-90cfe338325d.png)

### Please explain how to reproduce the issue or (better) provide an example to do it.
I just used the example provided in the readme.md => with the ComponentParam and ChildId components.

**Comment by bbenezech** [comment]
For our Google overlord:

```
Warning: Failed prop type: The prop `story` is marked as required in `StoryRouter`, but its value is `undefined`. in StoryRouter
```

**Comment by CoreyAR** [comment]
Looks like this issue is resolved in PR #44 https://github.com/gvaldambrini/storybook-router/pull/44

The PR removes the `story` prop which looks correct to me. When `StoryRouter` is rendered, the `story` is rendered as a child. The `children` prop is still included on `StoryRouter`. It never takes a `story` prop.

**Comment by gvaldambrini** [comment]
Yes, sorry for the delay. This is a regression introduced with #40. I'll try to publish a new version this week (hopefully later).
